mkosi/tree.py

```diff
@@ -76,7 +76,7 @@ def copy_tree(
         "cp",
         "--recursive",
         "--dereference" if dereference else "--no-dereference",
-        f"--preserve=mode,timestamps,links,xattr{',ownership' if preserve_owner else ''}",
+        f"--preserve=mode,timestamps,links{',ownership,xattr' if preserve else ''}",
```

Comment:
The xattr change should probably also be mentioned in the NEWS.

Reply:
I think we can probably get away without mentioning this. I'm 99% sure nobody relies on this.